plugins/repoquery.py

```diff
@@ -205,6 +212,8 @@ def run(self, args):
             q = q.filter(reponame=opts.repoid)
         if opts.arch:
             q = q.filter(arch=opts.arch)
+        if opts.file:
+            q = self.who_provides(opts.file, q)
```

Comment:
Any reason why you added extra one liner method? Can you put there just `query.filter(file=opts.file)` as in the options above, please? Otherwise looks good to me.